index.bs

```diff
@@ -65,16 +65,19 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
+The device orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
 
-Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
-of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
+The device orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
+of the [=low-level=] motion sensors (accelerometer, gyroscope). The type of fusion (complementary
```

Comment:
I cannot agree more and stressed the same in a meeting today :-) Now I believe what we wanted to spec first was the sensor for Web VR use-cases which would be the "Relative Orientation Sensor aka Game Orientation Sensor" - as it can be used for controlling stuff in things other than games, I like the former name, and think that we should rename this spec.

It is also easier to spec than the "Geographic" sensor which can apparently also use gyroscope in some cases which I don't understand the use-case for, as well as might expose declination etc with input from GPS.